tests/unit/backend/corpora/common/entities/__init__.py

```diff
@@ -0,0 +1,2 @@
+def get_ids(iterable):
```

Comment:
Would prefer this to be in a utility class rather than hidden in the `__init__.py` file.

Reply:
+1

Reply:
moved to utils.py